src/current/v25.1/changefeed-monitoring-guide.md

```diff
@@ -0,0 +1,168 @@
+        - Investigation needed: If checkpointing falls too far behind the current time.
+- Impact:
+    - Slow processing of changes and updates to downstream sinks.
+    - Recovery time after failures, potential for increase in change backlog.
+    - Ability to resume from last known good state.
+    - Resource usage during catch-up after restarts or failures.
+
+### End-to-end performance metrics
+
+#### End-to-end commit latency
+
+- Metric: `changefeed.commit_latency`
+- Description: The time between when a [transaction]({% link {{ page.version.version }}/transactions.md %}) commits and when the changefeed emits the corresponding changes. This time includes rangefeed and buffer delays.
+- Use with [metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels).
+
+#### Batch latency
+
+- Metric: `changefeed.sink_batch_hist_nanos`
+- Description: The end-to-end latency from the change event being put in a batch to the changefeed emitting the event message to the sink.
+- Investigation needed: If this metric has consistently high values, it indicates that there are bottlenecks in the pipeline.
+
+#### Protected timestamp age
+
+- Metric: `jobs.changefeed.protected_age_sec`
+- Description: The age of the [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps) record for the changefeed.
+- Investigation needed: If this metric has high values, it may indicate changefeed processing delays or resource constraints.
+- Impact: High values can [block garbage collection]({% link {{ page.version.version }}/protect-changefeed-data.md %}) and lead to increased disk usage and degraded performance for some workloads.
+
+#### Progress tracking
+
+- Metrics: 
+    - `high_water_timestamp` from [`SHOW CHANGEFEED JOBS`]({% link {{ page.version.version }}/show-jobs.md %}#show-changefeed-jobs) 
+    - `(now() - changefeed.aggregator_progress)`
+        - Use with [metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels).
+    - `changefeed.max_behind_nanos`
+        - Use with [metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels) (supported in v24.3.5+).
+    - `(now() - changefeed.checkpoint_progress)`
+- Description: These metrics help to measure how far behind the changefeed is from the current time.
+- Investigation: A growing delay indicates changefeed processing and messages emitting cannot keep up with the rate of change events.
+
+## Pipeline section metrics
+
+### Rangefeed
+
+#### Buffer memory usage
+
+- Metric: `changefeed.buffer_entries.allocated_mem.rangefeed`
+- Description: The current quota pool memory allocation between the rangefeed and the KV feed.
+- Impact: High memory usage may indicate backpressure.
+- Supported versions: v23.2.13+, v24.1.6+, v24.2.4+, v24.3.0+
```

Comment:
i think this supported versions info is not relevant for v25.1 docs?  e.g. users of v24.1 should have this info in those docs